pennylane/_qubit_device.py

```diff
@@ -183,6 +183,23 @@ def execute(self, circuit, **kwargs):
 
         return self._asarray(results)
 
+    def batch_execute(self, circuits, **kwargs):
```

Comment:
Gosh the `QubitDevice` version is so much cleaner

Reply:
I know, that's why I am so pushy when it comes to finally prioritising the refactor :)